docs/quickstart.rst

```diff
@@ -133,59 +217,61 @@ states it has assigned them:
 
 .. code-block:: text
 
+   # on the monitor virtual machine
+
+   sudo su - postgres
+   export PATH="/usr/lib/postgresql/11/bin/:$PATH"
+
```

Comment:
I like to export PGDATA here, rather than the PATH. In any case, both of those could be added to the profile of the user that is being used, or maybe that would be another extra step you'd rather bypass in the tutorial?